dev/conductor/README.md

```diff
@@ -23,6 +23,8 @@ unlock your ssh key for the session; if you do not, each time the conductor
 attempts to interact with a remote, the user will be prompted to enter their ssh
 key password.
 
+** Confirm that your personal github clone of flutter/flutter is named flutter and flutter/engine is named engine. If not you will need to use flags to overide the defaults.
+
```

Comment:
Should we remove all reference to the `--engine-cherrypicks` and `--framework-cherrypicks` options, since they're obsolete, and can lead to confusion?

Reply:
Maybe in the source we could mark the options deprecated too

Reply:
Since I didnt use the cherry picks flags and dont know how they work, I just am not qualified to say.

Reply:
please remove all references in the readme.md to these flags, and mark them deprecated in the source.

Reply:
removed reference in setup documentation. Will modify code in a different follow up pr.